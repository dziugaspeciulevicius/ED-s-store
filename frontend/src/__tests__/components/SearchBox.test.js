import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import SearchBox from "../../components/SearchBox.component";
jest.mock("../../components/SearchBox.component");

describe("SearchBox tests", () => {
  it("Renders <SearchBox /> component correctly", () => {
    // shallow creates instance of a component
    expect(shallow(<SearchBox />)).toMatchSnapshot();
  });
});
