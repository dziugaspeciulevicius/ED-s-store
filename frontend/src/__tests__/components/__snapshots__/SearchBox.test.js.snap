// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox tests Renders <SearchBox /> component correctly 1`] = `ShallowWrapper {}`;
